newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -78,7 +78,12 @@ export const OnlineGamePropertiesDialog = ({
         orientation: orientation || 'default',
       })
     ) {
-      await onApply({ discoverable });
+      await onApply({
```

Comment:
See my other comment, I would rather not specify things that are not changing rather than having this feeling when reading the code that it's going to set them to null.